rfcs/feature-set-metadata-file-to-web-platform-tests.md

```diff
@@ -0,0 +1,221 @@
+# RFC #163: Add feature_set metadata file to web-platform-tests
+
+Author: @jcscottiii
+
+# Introduction
+
+The web-platform-tests (WPT) project is a valuable resource for testing web
+platform features and contains valuable metadata such as feature specs.
+Recently, the [WebDX Community Group](https://www.w3.org/community/webdx/)
+began creating the [feature-set](https://github.com/web-platform-dx/feature-set)
+repository. That repository serves as a basic shared catalog of feature
+definitions of the web platform. Feature-set itself doesn't intend to produce
+data but rather link to existing data that will inform audiences which features
+are part of [Baseline](https://web.dev/baseline/). However, there's a great
+opportunity to connect the WPT ecosystem to the feature-set catalog. By doing
+this, it would enable users of wpt.fyi to filter by feature-set grouping, which
+is similar to
+[the ability to filter by spec](https://github.com/web-platform-tests/wpt.fyi/issues/1489).
+The RFC proposes the addition of a FEATURE_SET.yml metadata file which would
+enable the linkage between WPT and feature-set.
+
+# Proposed change
+
+The proposed change includes:
+
+1. Introduce a new metadata file, FEATURE_SET.yml
+2. Adjust the wpt-pr-bot to handle reviews of the changes
+3. Create a script to generate a manifest
+
+Below are the details for each step.
+
+## Step 1. Introduce a new metadata file, FEATURE_SET.yml
+
+Originally, there was a RFC that proposed adding these details to the existing
+META.yml. During the August 01, 2023 Monthly WPT
+[Meeting](https://github.com/web-platform-tests/wpt-notes/blob/master/minutes/2023-08-01.md#rfc-157---add-feature-meta-tag-to-web-platform-tests-meta-data),
+it was discussed that it should be a separate file. This section describes the structure of the file.
+
+This file is expected to be in the same places developers would expect a META.yml.
+
+### File examples
+
+```
+parser_version: v1
```

Comment:
Do we need this?

Reply:
Technically, no. But I'd rather be safe than sorry. I'll remove this and we can always add it back if needed.